bin/qless-js.js

```diff
@@ -60,6 +60,20 @@ const put = (name, options) => {
   logWithClient(options.parent, client => client.queue(name).put(options));
 };
 
+/*
+ * Recur a job
+ */
+const recur = (name, next, options) => {
+  logWithClient((client) => {
+    const timeNow = now();
```

Comment:
I don't see `now` defined anywhere? I assume you tested it and it works so I assume I'm just missing it

Reply:
Copypasta